### DISCUSSION TITLE
Image upload

### DISCUSSION THREAD

**Discussion by flashtheman** [standardnotes/forum#2626] [Suggestions] [answered]
Is there some native image upload planned for standard notes? I would like to copy/paste images from the clipboard natively like in other applications. In my opinion it's very cumbersome currently.

**Comment by JaspalSuri** [comment]
Something like it is planned, but there's no ETA. Currently the dev team is working on version 3.5, which will allow for the implementation of features like that in future releases. 

Transferring this to the forum since it could apply to our mobile apps as well.

**Comment by arunsathiya** [comment]
@JaspalSuri Would that still remain an Extended feature, or are we discussing about bringing that feature to free plan as well?

**Comment by JaspalSuri** [comment]
I don't know how/if it will play out into both plans just yet, but I'll discuss it with the team for consideration!

**Comment by myreli** [comment, marked as answer]
It's available since `v3.20` with the release of [Encrypted Storage](https://standardnotes.com/blog/introducing-encrypted-file-storage-with-automated-backups).
